loops-and-arrays.js

```diff
@@ -5,6 +5,9 @@
 // ==========================================
 
 const names = ["Henk", "Piet", "Fred", "Joop"];
+for ( let i = 0; i < names.length; i++) {
+    console.log (names[i] +"je");
+}
 
```

Comment:
In deze opdracht word gevraagd om de namen in de array te vervangen door de koosnaampjes. Deze for loop print wel de juiste koosnaampjes uit maar past de array zelf niet aan. consol.log(names) geeft nog steeds ["Henk", "Piet", "Fred", "Joop"] en dit zou ["Henkje", "Pietje", "Fredje", "Joopje"] moeten zijn. Misschien kan je dat nog aanpassen.